Cargo.toml

```diff
@@ -3,46 +3,53 @@ resolver = "2"
 members = ["crates/*"]
 
 [workspace.dependencies]
-anyhow = "1"
-async-trait = "0.1"
+anyhow = "1.0.82"
+async-trait = "0.1.80"
+axum = "0.6"
```

Comment:
Updating `axum` is a rabbit hole, but we need to do it sooner or later. The reason is that in order to update `axum` we need to update altogether:
- `hyper`
- every `http` crate
- `reqwest`
- `warp` (we are using our own forked version 3 years old...)

I'd recommend to fully delete `warp` dependency (latest versions of `axum` basically do everything which `warp` does anyway). And then update `axum` + `hyper` which both have decent breaking changes.

What do you guys think? should I create a task for it since it will be tricky to do?

I need to do this sooner or later, because we cannot update any of those crates.